packages/action-text-field/action-text-field.scss

```diff
@@ -0,0 +1,21 @@
+@import "~@fremtind/jkl-core/variables/_all.scss";
+
+.jkl-icon-text-field {
+    display: block;
+    position: relative;
+
+    &__field {
+        .jkl-text-field__input {
+            padding-right: 2.5rem;
```

Comment:
I believe `$component-spacing--xxl` can be used here for the same value :) Alternatively (and preferrably?) you should define the size of the icon somewhere with a variable, and define this padding using that variable and a spacing-unit. E.g.
```scss
$icon-size: rem(20px); // or something else
// ...
.jkl-text-field__input {
    padding-right: $icon-size + $component-spacing--small;
}
```

Makes it easoer to deduce the reason for the value.